crates/pessimistic-proof/src/global_index.rs

```diff
@@ -0,0 +1,89 @@
+use reth_primitives::U256;
+use serde::{Deserialize, Serialize};
+
+/// The [`GlobalIndex`] uniquely references one leaf within one Global Exit Tree.
+/// Further defined by the LXLY specifications.
+/// | 191 bits |    1 bit      |    32 bits   |      32 bits     |
+/// |    0     |  mainnet flag | rollup index | local root index |
```

Comment:
nit: I find `leaf index` (as in the struct) a bit clearer than `local root index`

Reply:
Renamed :+1: